package.json

```diff
@@ -30,6 +30,7 @@
     "ip3country": "^5.0.0",
     "node-fetch": "^2.6.1",
     "object-hash": "^2.1.1",
+    "redis": "^4.3.0",
```

Comment:
hmm this definitely needs to get split out then.  Our core node SDK should not need to take a dependency on redis, only this adapter.